warmup.py

```diff
@@ -240,17 +251,39 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        for i, point in enumerate(points[:-1]):
+            neighbors = np.array(points[i+1:])
+            for neighbor in neighbors:
```

Comment:
This is the right idea, but you probably don't want to implement it like this because it gives NumPy the option (and it probably does) make a copy of points. Instead, I would loop over an index `j` using `range()`.